apps/journeys-admin/src/components/Editor/Slider/Settings/Drawer/ImageBlockThumbnail/ImageBlockThumbnail.tsx

```diff
@@ -44,28 +62,12 @@ export function ImageBlockThumbnail({
       ) : selectedBlock?.src != null ? (
         <Box
           component="img"
-          srcSet={
-            // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-            isUnsplash
-              ? `${selectedBlock.src
-                  .replace('w=1080', 'w=55&h=55&auto=format&dpr=2')
-                  .replace('fit=max', 'fit=crop')
-                  .replace('crop=entropy&', '')} 2x`
-              : undefined
-          }
-          src={
-            // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-            isUnsplash
-              ? selectedBlock.src
-                  .replace('w=1080', 'w=55&h=55&auto=format')
-                  .replace('fit=max', 'fit=crop')
-                  .replace('crop=entropy&', '')
-              : selectedBlock.src
-          }
+          srcSet={getImageSource(true)}
+          src={getImageSource(false)}
```

Comment:
Might be cleaner to have two different functions 
```suggestion
          srcSet={getImageSourceSet()}
          src={getImageSource()}
```

And have a function called `parseUnsplash(selectedBlock.src)` so something like...
```
getImageSource(){
  return parseUnsplash(selectedBlock.src)
}

getImageSourceSet(){
  return `${parseUnsplash(selectedBlock.src.replace('w=1080', 'w=1080&dpr=2'))} 2x`
}
```

What do you think?